### PR TITLE
eliminate dead initializations

### PR DESCRIPTION
**Pull request by jserv**

### CODE REVIEW

File: pm_serde_example/pm_read.c [hunk 1]
```diff
@@ -8,7 +8,6 @@ int main (int argc, char** argv)
 {
   OPObject* list;
   FILE* in = fopen("list_serialized", "r");
-  PMMemoryManager* ctx = PMDeserialize(in, &list);
```
Comment (dryman) [inline pm_serde_example/pm_read.c]: Without this line program wouldn't work.
We should free the manager in the end instead.

Comment (dryman) [inline pm_serde_example/pm_read.c]:
```c
PMMemoryManager_destroy(ctx);
```
At the end of the program.

Comment (jserv) [inline pm_serde_example/pm_read.c]: Done!

**Comment by dryman** [comment]
Closing old PR.

**Pull request closed without merging**